From 59715fdefaa56b253c4971d101d4765c1ece2236 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcus=20M=C3=BCller?= <mmueller@gnuradio.org>
Date: Tue, 11 Aug 2026 23:02:36 +0200
Subject: [PATCH] tests: use correct fmt include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

On current fmt (12.2 fmt, probably starting earlier), fmt/core.h does no
longer include fmt::format.

Signed-off-by: Marcus Müller <mmueller@gnuradio.org>

Upstream-Status: Backport
---
 lib/qa_utils.cc                                | 2 +-
 tests/test_volk_32f_x3_sum_of_poly_32f.cc      | 2 +-
 tests/test_volk_32fc_s32fc_x2_rotator2_32fc.cc | 2 +-
 tests/test_volk_32fc_x2_multiply_32fc.cc       | 2 +-
 tests/volk_test.cc                             | 2 --
 tests/volk_test.h                              | 2 +-
 6 files changed, 5 insertions(+), 7 deletions(-)

diff --git a/lib/qa_utils.cc b/lib/qa_utils.cc
index b336c1d..c635f3e 100644
--- a/lib/qa_utils.cc
+++ b/lib/qa_utils.cc
@@ -28,7 +28,7 @@
 #include <random>
 #include <vector> // for vector, _Bit_refe...
 
-#include <fmt/core.h>
+#include <fmt/format.h>
 
 // Warmup time for CPU frequency scaling (ms)
 static double g_warmup_ms = 2000.0;
diff --git a/tests/test_volk_32f_x3_sum_of_poly_32f.cc b/tests/test_volk_32f_x3_sum_of_poly_32f.cc
index 282cffc..74a19db 100644
--- a/tests/test_volk_32f_x3_sum_of_poly_32f.cc
+++ b/tests/test_volk_32f_x3_sum_of_poly_32f.cc
@@ -9,7 +9,7 @@
 
 #include "volk_test.h"
 #include <fmt/chrono.h>
-#include <fmt/core.h>
+#include <fmt/format.h>
 #include <fmt/ranges.h>
 #include <gtest/gtest.h>
 #include <volk/volk.h>
diff --git a/tests/test_volk_32fc_s32fc_x2_rotator2_32fc.cc b/tests/test_volk_32fc_s32fc_x2_rotator2_32fc.cc
index f899657..cbd241e 100644
--- a/tests/test_volk_32fc_s32fc_x2_rotator2_32fc.cc
+++ b/tests/test_volk_32fc_s32fc_x2_rotator2_32fc.cc
@@ -9,7 +9,7 @@
 
 #include "volk_test.h"
 #include <fmt/chrono.h>
-#include <fmt/core.h>
+#include <fmt/format.h>
 #include <fmt/ranges.h>
 #include <gtest/gtest.h>
 #include <volk/volk.h>
diff --git a/tests/test_volk_32fc_x2_multiply_32fc.cc b/tests/test_volk_32fc_x2_multiply_32fc.cc
index e700314..04c183f 100644
--- a/tests/test_volk_32fc_x2_multiply_32fc.cc
+++ b/tests/test_volk_32fc_x2_multiply_32fc.cc
@@ -9,7 +9,7 @@
 
 #include "volk_test.h"
 #include <fmt/chrono.h>
-#include <fmt/core.h>
+#include <fmt/format.h>
 #include <fmt/ranges.h>
 #include <gtest/gtest-param-test.h>
 #include <gtest/gtest.h>
diff --git a/tests/volk_test.cc b/tests/volk_test.cc
index bf9a991..8f1829f 100644
--- a/tests/volk_test.cc
+++ b/tests/volk_test.cc
@@ -7,8 +7,6 @@
  * SPDX-License-Identifier: LGPL-3.0-or-later
  */
 
-#include <fmt/core.h>
-#include <fmt/ranges.h>
 #include <gtest/gtest.h>
 #include <volk/volk.h>
 #include <algorithm>
diff --git a/tests/volk_test.h b/tests/volk_test.h
index 1298bef..5ec640b 100644
--- a/tests/volk_test.h
+++ b/tests/volk_test.h
@@ -7,7 +7,7 @@
  * SPDX-License-Identifier: LGPL-3.0-or-later
  */
 
-#include <fmt/core.h>
+#include <fmt/format.h>
 #include <fmt/ranges.h>
 #include <gtest/gtest.h>
 #include <volk/volk.h>
-- 
2.55.0

